creator/ingest_runs/manifests.py

```diff
@@ -1,334 +0,0 @@
-    all_four = get_biospecimen_sequencing_experiment_links(study_id)
-    # Join genomic_df and all_four on biospecimen ID, then load
-    assert CONCEPT.BIOSPECIMEN.ID in genomic_df
-    assert CONCEPT.BIOSPECIMEN.ID in all_four
-    genomic_seq_df = genomic_df.merge(
-        right=all_four,
-        on=CONCEPT.BIOSPECIMEN.ID,
-        how='inner',
-    )
-    df = genomic_seq_df[[
-        CONCEPT.GENOMIC_FILE.TARGET_SERVICE_ID,
-        CONCEPT.SEQUENCING.ID,
-        CONCEPT.GENOMIC_FILE.ID,
-        CONCEPT.SEQUENCING.CENTER.TARGET_SERVICE_ID,
-    ]]
-    df.drop_duplicates(inplace=True)
-    df[CONCEPT.SEQUENCING.TARGET_SERVICE_ID] = df[CONCEPT.SEQUENCING.ID]
-    df[CONCEPT.SEQUENCING_GENOMIC_FILE.VISIBLE] = True
-
-    # Load the genomic-file sequencing-experiment links
-    _ = load_entities(SEQ_EXP_GEN_FILE, df, study_id)
-    return df
-
-
-def get_metadata(manifest_df):
-    """
-    Scrape the metadata for each file in the manifest from S3, put it into a
-    DataFrame, join this with _manifest_df_ on Filepath, and return it.
-    """
-    assert not manifest_df.empty, "Empty manifest DataFrame"
-    assert "Filepath" in manifest_df.columns, "Missing Filepath column"
-
-    buckets = set(os.path.split(row)[0] for row in manifest_df["Filepath"])
-    file_df = scrape_s3(buckets)
-
-    genomic_df = manifest_df.merge(right=file_df, on="Filepath", how="inner")
-    return genomic_df
-
-
-def scrape_s3(buckets):
-    """
-    Perform the scrape for each bucket in _buckets_. Return a DataFrame _df_
-    which has all this data.
-    """
-    dfs = []
-    for bucket in buckets:
-        parsed_s3 = urlparse(bucket)
-        bucket_df = pd.DataFrame(
-            fetch_bucket_obj_info(
-                bucket_name=parsed_s3.netloc,
-                search_prefixes=parsed_s3.path,
-                drop_folders=True,
-            )
-        )
-        bucket_df["Filepath"] = f"s3://{parsed_s3.netloc}/" + bucket_df["Key"]
-        dfs.append(bucket_df)
-
-    df = pd.concat(dfs)
-    df["Filename"] = df["Key"].map(lambda x: os.path.split(x)[-1])
-    return df
-
-
-def load_harmonized_genomic_files(genomic_df, study_id):
-    """
-    Load data for the harmonized genomic files found in the manifest into the
-    DataService (this data is in _genomic_df_).
-    This step must occur before the others so that we can use the
-    kf_ids created here for the rest of the process. After ingestion, grab the
-    generated kf_ids from the ingest lib cache, join them to _genomic_df_.
-    Then use _genomic_df_ to create genomic-file biospecimen links and load
-    them as well. Return _genomic_df_.
-    """
-    genomic_df = munge_genomic_df(genomic_df)
-    genomic_cache = load_entities(
-        GEN_FILE,
-        genomic_df.drop(CONCEPT.BIOSPECIMEN.ID, axis=1),
-        study_id,
-    )
-    genomic_df = get_genomic_file_kf_ids(genomic_cache, genomic_df)
-    
-    biospec_gen_df = genomic_df[[
```

Comment:
I liked that you seemed to have a load_<entity type>_  method for each type, so I pulled out the loading of the biospecimen-genomic-files into its own load method too